frontend/src/app/reducers/addon.js

```diff
@@ -204,16 +181,13 @@ export default function addonReducer(state: ?AddonState, action: AddonActions):
       return setInstalledAddons(state, action);
     case "SET_CLIENT_UUID":
       return setClientUuid(state, action);
+    case "EXPERIMENT_INSTALLED":
+      cookies.set("exp-installed", action.payload.addon_id);
```

Comment:
yeah, reducers aren't supposed to have side effects but these seem fairly benign and I'd prefer to keep cookies out of InstallManager.